pensjon-brevbaker/src/main/kotlin/no/nav/pensjon/brev/maler/fraser/common/Felles.kt

```diff
@@ -101,19 +101,21 @@ object Felles {
         }
     }
 
-    val kroner = TextOnlyPhrase<LangBokmalNynorskEnglish, Kroner> { kroner ->
-        textExpr(
-            Bokmal to kroner.format() + " kr",
-            Nynorsk to kroner.format() + " kr",
-            English to kroner.format() + " NOK",
-        )
+    data class KronerText(val kroner: Expression<Kroner>) : TextOnlyPhrase2<LangBokmalNynorskEnglish>() {
+        override fun TextOnlyScope<LangBokmalNynorskEnglish, Unit>.template() =
+            textExpr(
+                Bokmal to kroner.format() + " kr",
+                Nynorsk to kroner.format() + " kr",
+                English to kroner.format() + " NOK",
```

Comment:
Har ikke engelsk oftest NOK før beløpet?
You will recieve NOK 123.

Reply:
(Trenger ikke å være en del av denne PRen)

Reply:
Tja, jo. Vet ikke helt.

Reply:
Kan avvente en faglig vurdering :)